src/Sulu/Bundle/ContactBundle/Resources/views/Template/base.form.html.twig

```diff
@@ -38,7 +38,7 @@
                     <script type="text/template" id="address-tpl">
                         <div class="grid-col-12 floating m-top-0 address-component">
                             <~ var uid = _.uniqueId('address_'), first = true; ~>
-                            <~ var getAddressString=function(street,number,zip,city,country){var str=!!street?street:"",part=(zip+" "+city).trim();str+=!!str.length&&!!number?" "+number:number;str+=!!str.length&&!!part?", "+part:part;str+=!!str.length&&!!country?", "+country:country;return str}; ~>
+                            <~ var getAddressString=function(title,street,number,zip,city,country){var str=(!!title?title:""),part=(zip+" "+city).trim();str+=(!!street?(!!str.length?", ":"") + street:"");str+=!!str.length&&!!number?" "+number:number;str+=!!str.length&&!!part?", "+part:part;str+=!!str.length&&!!country?", "+country:country;return str}; ~>
```

Comment:
@chirimoya i also dont like this but i have tried following:

````
<~ var getAddressString = function(title, street, number, zip, city, country) {
var str = (!!title ? title : ""),
    part = (zip + " " + city).trim();

    str += (!!street ? (!!str.length ? ", " : "") + street : "");
    str += !!str.length && !!number ? " " + number : number;
    str += !!str.length && !!part ? ", " + part : part;
    str += !!str.length && !!country ? ", " + country : country;

    return str;
}; ~>
````

and this does not because of this error:

![image](https://cloud.githubusercontent.com/assets/1464615/11175842/0e5b7fd0-8c37-11e5-9d68-0771724df86b.png)

so it looks like that underscore template cannot handle multiline expressions ... or do you see an error in my code?